src/content/docs/browser/new-relic-browser/guides/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -5,8 +5,6 @@ tags:
   - Browser monitoring
   - Guides
 metaDescription: "A guide to monitor Google's core web vitals."
-redirects:
-  - /docs/browser/new-relic-browser/guides/guide-to-core-web-vitals
```

Comment:
@jeff-colucci How come this is removed? I think this URL is in the UI somewhere, so I think we should keep it as a redirect.

Reply:
I'm actually not sure! @bradleycamacho, did we remove this one on purpose?